src/SDKs/StorageSync/Management.StorageSync/AzSdk.RP.props

```diff
@@ -0,0 +1,7 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
@ankushbindlish2 remove this file, this seems redundant

Reply:
I see it in other resource providers. Please let me know if it is ok to keep it.